ssds/blobstore/gs.py

```diff
@@ -48,6 +48,9 @@ def put(self, bucket_name: str, key: str, data: bytes):
         blob = _client().bucket(bucket_name).blob(key)
         blob.upload_from_file(io.BytesIO(data))
 
+    def cloud_native_checksum(self, bucket_name: str, key: str):
```

Comment:
I think this should be `    def cloud_native_checksum(self, bucket_name: str, key: str) -> str:` based on the baseclass.

Reply:
will fix